Index: server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/locators/SingleBrowserLocator.java
===================================================================
--- server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/locators/SingleBrowserLocator.java	(revision 2478)
+++ server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/locators/SingleBrowserLocator.java	(working copy)
@@ -144,8 +144,7 @@
         }
 
         if (LauncherUtils.isScriptFile(launcher)) {
-            LOGGER.warn("Ignoring '" + launcher.getAbsolutePath() +"': file is a script file, not a real executable");
-            return null;
+            LOGGER.warn("Caution: '" + launcher.getAbsolutePath() +"': file is a script file, not a real executable.  The browser environment is no longer fully under RC control");
         }
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("Discovered valid " + browserName() + " launcher  : '" + launcher + "'");
Index: server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/BrowserLauncherFactory.java
===================================================================
--- server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/BrowserLauncherFactory.java	(revision 2478)
+++ server-coreless/src/main/java/org/openqa/selenium/server/browserlaunchers/BrowserLauncherFactory.java	(working copy)
@@ -48,11 +48,10 @@
         supportedBrowsers.put("chrome", FirefoxChromeLauncher.class);
         supportedBrowsers.put("firefox2", Firefox2Launcher.class);
         supportedBrowsers.put("firefox3", Firefox3Launcher.class);
-        supportedBrowsers.put("iexploreproxy", InternetExplorerCustomProxyLauncher.class);
+        supportedBrowsers.put("iexplore", InternetExplorerCustomProxyLauncher.class);
         supportedBrowsers.put("safari", SafariFileBasedLauncher.class);
         supportedBrowsers.put("safariproxy", SafariCustomProfileLauncher.class);
         supportedBrowsers.put("iehta", HTABrowserLauncher.class);
-        supportedBrowsers.put("iexplore", HTABrowserLauncher.class);
         supportedBrowsers.put("opera", OperaCustomProfileLauncher.class);
         supportedBrowsers.put("piiexplore", ProxyInjectionInternetExplorerCustomProxyLauncher.class);
         supportedBrowsers.put("pifirefox", ProxyInjectionFirefoxCustomProfileLauncher.class);
