Index: source/data/mappings/windows-936-2000.ucm
===================================================================
--- source/data/mappings/windows-936-2000.ucm	(revision 7525)
+++ source/data/mappings/windows-936-2000.ucm	(working copy)
@@ -260,6 +260,7 @@
 <U01DA> \xA8\xB7 |0
 <U01DB> \xA8\xB8 |1
 <U01DC> \xA8\xB8 |0
+<U01F9> \xA8\xBF |1
 <U0251> \xA8\xBB |0
 <U0261> \xA8\xC0 |0
 <U02C7> \xA1\xA6 |0
@@ -381,6 +382,7 @@
 <U044E> \xA7\xF0 |0
 <U044F> \xA7\xF1 |0
 <U0451> \xA7\xD7 |0
+<U1E3F> \xA8\xBC |1
 <U2010> \xA9\x5C |0
 <U2013> \xA8\x43 |0
 <U2014> \xA1\xAA |0
@@ -473,6 +475,7 @@
 <U2299> \xA1\xD1 |0
 <U22A5> \xA1\xCD |0
 <U22BF> \xA8\x53 |0
+<U22EF> \xA1\xAD |1
 <U2312> \xA1\xD0 |0
 <U2460> \xA2\xD9 |0
 <U2461> \xA2\xDA |0
@@ -697,6 +700,7 @@
 <U3015> \xA1\xB3 |0
 <U3016> \xA1\xBC |0
 <U3017> \xA1\xBD |0
+<U301C> \xA1\xAB |1
 <U301D> \xA8\x94 |0
 <U301E> \xA8\x95 |0
 <U3021> \xA9\x40 |0
